dts/bindings/pinctrl/st,stm32-pinmux.yaml

```diff
@@ -11,6 +11,45 @@ properties:
     reg:
       required: true
 
-pinmux-cells:
-  - pin
-  - function
+child-binding:
```

Comment:
Needs to have a `title:` and `description:` too. The contents of `child-binding:` is parsed like a regular binding.

Reply:
@ulfalizer 
The usage of child-binding did not work out for me with the current edtlib.py.

In #22255 I made a RFC for a different solution using a generic binding for the pinctrl-state nodes (and some other similar node types - dts: edtlib: enable bindings for special nodes without compatibles).